Frameworks/CoreGraphics/CGBitmapContext.mm

```diff
@@ -45,12 +48,13 @@ CGImageRef CGBitmapContextCreateImage(CGContextRef context) {
 }*/
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes tied to underlying implementation caveats in CGImageGetBitmapInfo. 
 */
 CGBitmapInfo CGBitmapContextGetBitmapInfo(CGContextRef context) {
-    UNIMPLEMENTED();
-    return StubReturn();
+	CGImageRef imageRef = context->Backing()->DestImage();
```

Comment:
Could you run tools/scripts/clang-format.ps1 on these files?  It will unify tabs/formatting